src/components/SvgIcon/assets/medicalclipboard.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M9 2a1 1 0 0 0-1 1v1a1 1 0 0 0 1 1h6a1 1 0 0 0 1-1V3a1 1 0 0 0-1-1H9ZM5.063 5.041c.032-.034.062-.041.08-.041h1.143a1 1 0 1 0 0-2H5.143a2.11 2.11 0 0 0-1.536.67C3.212 4.09 3 4.646 3 5.214v14.572c0 .568.212 1.124.607 1.544.396.421.947.67 1.536.67h13.714a2.11 2.11 0 0 0 1.537-.67c.394-.42.606-.976.606-1.544V5.214c0-.568-.212-1.124-.607-1.544A2.11 2.11 0 0 0 18.857 3h-1.143a1 1 0 1 0 0 2h1.143c.018 0 .048.007.08.041a.255.255 0 0 1 .063.173v14.572a.255.255 0 0 1-.063.173c-.032.034-.062.041-.08.041H5.143c-.018 0-.048-.007-.08-.041A.255.255 0 0 1 5 19.786V5.214c0-.075.029-.137.063-.173ZM11.7 8.13a1 1 0 0 1 1 1v2.745h2.8a1 1 0 1 1 0 2h-2.8v2.755a1 1 0 0 1-2 0v-2.755H8a1 1 0 1 1 0-2h2.7V9.13a1 1 0 0 1 1-1Z" fill="#6C737A"/></svg>
```

Comment:
I think it is good idea to remove fill in SVG, it  will make possible to change colors of icons from CSS, using fill and make svg more reusable and generic.

Reply:
Just like here https://github.com/EvgeniyKvasyuk/MAHE_WP/pull/31/files#diff-73249ea5e365f14a1684c44242f3e8346670060f9e0cc3522f358e9fa77bf22b

Reply:
Updated The PR